Source/Core/DolphinWX/ISOProperties.cpp

```diff
@@ -1296,9 +1304,16 @@ void CISOProperties::OnComputeMD5Sum(wxCommandEvent& WXUNUSED (event))
 	m_MD5Sum->SetValue(output_string);
 }
 
+// Opens all pre-defined INIs for the game. If there are multiple ones,
+// they will all be opened, but there is usually only one
 void CISOProperties::OnShowDefaultConfig(wxCommandEvent& WXUNUSED (event))
 {
-	LaunchExternalEditor(GameIniFileDefault);
+	for (const std::string& filename : SCoreStartupParameter::GetGameIniFilenames(game_id, OpenISO->GetRevision()))
+	{
+		std::string path = File::GetSysDirectory() + GAMESETTINGS_DIR DIR_SEP + filename;
+		if (File::Exists(path))
+			LaunchExternalEditor(path);
```

Comment:
Yes. Being aware of all the INIs is useful, because they can inherit properties from each other (but only SSBM actually uses this right now as far as I know), and because opening all the INIs helps avoidng forgetting updating one of the INIs. Most games will only have one INI, but games that have AR codes or patches will have two, so not a lot of extra editors will be opened. This is only true if issue 8258 is fixed, though...